### ISSUE TITLE
new account. how are server suggestions selected?

### ISSUE THREAD

**Issue by zz-ha-dum** [kriztan/Pix-Art-Messenger#251]
1
why is 404.city not shown first?  (404.city wonders why)

2
should only compliance.conversations.com 100% rated servers be shown?  why introduce needless problems to novice users?

3
an expert option to stop suggesting for servers on account creation AND contact adding would be much appreciated

**Comment by kriztan** [comment]
We have a cooperation with blabber.im provider, which is preselected for new users. But you have the choice to select another provider. 
The server list is mix from compliance list and third party users/prividers which wanted me to add their servers.

**Comment by Echolon** [comment]
#203 Feature change: Account registration server list - set Pix-Art.de as default

**Comment by MalevolentGrin** [comment]
> The server list is mix from compliance list and 


> third party users/prividers which wanted me to add their servers.

ohhh

> > maybe paranthetically display compliance rating with an explanation button?

That seems like a good mix addition:  allow people to know they might have a server that will have less than ideal capabilities.

**Comment by genofire** [comment]
https://compliance.conversations.im has an API : https://compliance.conversations.im/api/compliant_servers/ 
That is a list of all servers with 100% required services + the optional service In-Band registration (a perfect state for clients).

On this way is the serverlist inside of Pix-Art not needed and must not been updated / maintained.

**Comment by Echolon** [comment]
@genofire I disagree to add more servers. Newcomers have no clue how to decide about it. I'd rather suggest to reduce the number of servers. You still can register somewhere else and use another account if necessary

**Comment by ToEatFreshBabies** [comment]
Show only 100% compliant servers.  Win.

Consider an option to on-demand generate latency scores to help users narrow otherwise uninformed choices.

**Comment by Echolon** [comment]
@zz-ha-dum 

Is that issue cleared out for you?
